lib/ret/hub.ex

```diff
@@ -117,11 +119,13 @@ defmodule Ret.Hub do
     |> unique_constraint(:entry_code)
   end
 
-  def add_meta_to_changeset(changeset, attrs) do
+  def add_attrs_to_changeset(changeset, attrs) do
     changeset
-    |> cast(attrs, [:name, :description])
+    |> cast(attrs, [:name, :description, :member_cap])
     |> validate_required([:name])
     |> validate_length(:name, max: 64)
+    |> validate_length(:description, max: 64_000)
+    |> validate_number(:member_cap, greater_than_or_equal_to: 0, less_than_or_equal_to: 64)
```

Comment:
I'd argue to bump the max to 100

Reply:
Seems fine. Probably worth just exposing this to the admin console as well since we are going to expose a default anyway.